Delete outdated/unused messages and add/update CR/WAI/WF messages

JarlPenguin opened the pull request:
Changes in more detail:
* Delete Hashtag issue message
  * Superseded by various bug reports, and users cannot add labels to their reports anymore after the migration to Cloud
* Delete Auth server down message
  * Superseded by Temporary service outage
* Delete Legacy launcher version message
  * Legacy launcher is discontinued and unsupported
* Delete Panel - Mojang Priority message
  * Users cannot modify Mojang Priority anymore after the migration to Cloud
* Delete Modified world crash message
  * Superseded by Modified game
* Add Cannot Reproduce and Won't Fix messages
  * Adapted from bugnet variants
* Update Works As Intended messages
  * Works As Intended (internal) for internal purposes
  * Works As Intended for regular use
  * Adapted links from bugnet
  * Enable for all projects
* Also enable Attach world file and Pirated Minecraft messages for MCPE
* `steps-to-reproduce` -> `provide-steps-to-reproduce` for consistency
* `Temporary Service Outage` -> `Temporary service outage` for consistency

Greymagic27 commented on assets/messages.yml:
```diff
@@ -87,24 +69,37 @@ variables:
+  wai_links:
+    - project: bds
+      value: |-
+        [Full Version History|https://minecraft.wiki/w/Bedrock_Dedicated_Server#History] -- [The official Minecraft feedback site|https://feedback.minecraft.net]
```
We always say wiki is not a WAI source. Should we be linking to it when leaving messsages about WAI?

Greymagic27 replied:
Oh I see where it is placed now. We do have it in the current messages.

Wondering still if it's good to link to those in a WAI message though

JarlPenguin replied:
I wouldn't want to deviate from bugnet, but I'm not against this per se (see #87 also).

Greymagic27 replied:
Oh does even bugnet put this? I guess we keep it then to maintain parity and look at 87 down the line

Greymagic27 commented on assets/messages.yml:
```diff
@@ -87,24 +69,37 @@ variables:
+  wai_links:
+    - project: bds
+      value: |-
+        [Full Version History|https://minecraft.wiki/w/Bedrock_Dedicated_Server#History] -- [The official Minecraft feedback site|https://feedback.minecraft.net]
```
Wondering if we should link to discord here/instead.

The feature request message lists both

JarlPenguin replied:
Same as https://github.com/mojira/helper-messages/pull/249#discussion_r2049294960

Greymagic27 commented on assets/messages.yml:
```diff
@@ -257,25 +250,32 @@ messages:
-      shortcut: auth
-      category: invalid
+    - project:
+        - mcpe
```
Do we want to have BDS listed in the here as well? On line 240-241

JarlPenguin replied:
Done at https://github.com/mojira/helper-messages/pull/249/commits/932d43591fb60b4608f014a26c42508799bd31a0

Greymagic27 commented on assets/messages.yml:
```diff
@@ -318,9 +316,26 @@ messages:
+      message: |-
+        *Thank you for your report!*
+        ~Unfortunately, the described issue cannot be reproduced. Therefore, this ticket is being resolved as {color:#d04437}*Cannot Reproduce*{color}.~
+
```
Add something saying

"If you can reproduce this issue in the latest release and/or development snapshot please make a new bug report containing steps to reproduce"

Or something like that. Like how the incomplete mesasge does it

Greymagic27 commented on assets/messages.yml:
```diff
@@ -354,22 +369,6 @@ messages:
-        - mcd
-        - mclg
-      name: Crash logged automatically
-      shortcut: autocrash
```
How come this one is removed?

JarlPenguin replied:
MCD/MCLG are obsolete

Greymagic27 replied:
Oh.. right. From the diff it looked like it had been removed completely, not just the two projects

Greymagic27 commented on assets/messages.yml:
```diff
@@ -1295,22 +1168,23 @@ messages:
         However, this issue is {color:#FF5722}*Invalid*{color}.
 
-        You are currently using a *non-authorized* version of Minecraft. If you wish to purchase the full game, please visit the [Minecraft Store|https://www.minecraft.net/store/minecraft-java-edition].
+        You are currently using a *non-authorized* version of Minecraft. If you wish to purchase the full game, please visit the [Minecraft Store|https://www.minecraft.net/store/minecraft-deluxe-collection-pc].
```
Should we link to the normal version instead of deluxe, even if that is the default option when visiting that page?
https://www.minecraft.net/en-us/store/minecraft-java-bedrock-edition-pc

JarlPenguin replied:
Done

Greymagic27 commented on assets/messages.yml:
```diff
@@ -1295,22 +1168,23 @@ messages:
     - project:
         - mc
         - mcl
+        - mcpe
```
Is it worth adding WEB here? Sometimes people report pirated things there. It's rare, but can happen

JarlPenguin replied:
Wouldn't hurt for sure. I'm not sure myself what warrants a message not being enabled for a certain project if it is at least somewhat applicable to it.

Greymagic27 replied:
I'm happy to add it even if it's minimal usage. Doesn't do any harm and WEB doesn't have so many messages that it'd cause clutter either

JarlPenguin commented:
Post-opening changes:
* Enabled Attach world file message for BDS
* Enabled Pirated Minecraft, Temporary service outage, Multiple issues in one report, Feature request messages for WEB

Some more things to consider:
* Is the "Please do not unmark issues from _private_ after a moderator had set it that way." still relevant, since users can no longer change the security level after the creation of a report?

JarlPenguin commented:
* Deleted Unmarked private issue panel
* Updated Wrong project (Moved) message: https://github.com/mojira/helper-messages/pull/248/commits/cfbe525ed89d2e9729ed05134b1cb6af732459c5

JarlPenguin commented:
Updated Attach video message: https://github.com/mojira/helper-messages/pull/248/commits/8d7e62be3c1c0c557ca075c5504bc137855d19e9

Greymagic27 reviewed (approved):
Happy with all the changes
